js/symbol/glyph_atlas.js

```diff
@@ -115,9 +115,8 @@ GlyphAtlas.prototype.resize = function() {
     var origw = this.width,
         origh = this.height;
 
-    // For now, don't grow the atlas beyond 1024x1024 because of how
-    // texture coords pack into unsigned byte in symbol bucket.
-    if (origw > 512 || origh > 512) return;
+    // Grow atlas to 2048 for CJK characters
+    if (origw > 1024 || origh > 1024) return;
```

Comment:
This would be more readable if this magic number were pulled out into a constant and documented.